### PR TITLE
Changed email notification to address [OSF-5026]

### PR DESCRIPTION
**Pull request by njantrania**
<b>Purpose</b>

The purpose of this was to change the email sent to registered contributors when notifying them about being added to a project so that the email would say who added them.  

<b>Changes</b>

The change made was to add the name of the author of the project to the email so that the recipient could tell who added them.

### CODE REVIEW

File: website/templates/emails/contributor_added.txt.mako [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Hello ${user.fullname},
 
-You have been added as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
+${node.contributors[0]} has added you as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
```
Comment (samanehsan) [inline website/templates/emails/contributor_added.txt.mako]: I believe `node.contributors[0]` is the user who created the project, but any administrator on the project can add contributors. 

Can you pass in the name of the user who added the contributor to this email? I don't think the auth is being passed to `notify_added_contributor` so you'll need to do that in order to pass `auth.user.fullname` to the email template.

**Comment by samanehsan** [comment]
Pass complete @njantrania! :snowman:

**Comment by GaryKriebel** [comment]
@njantrania as I believe this is your last day, please let us know if you are unable to update this PR per the suggestions of @samanehsan.  Thanks.

**Comment by njantrania** [comment]
Unfortunately I can't - I've already reset my computer so I can turn it in.

**Comment by samanehsan** [comment]
@njantrania, I noticed your commits aren't being associated with your github account. Here's how to set up your email so that github knows the commits are yours and gives you credit for them: https://help.github.com/articles/setting-your-email-in-git/.

This isn't urgent, I just wanted to let you know for future github use!

**Pull request closed without merging**